Sources/Thumbprint/Presentations/Sheet/PartialSheetPresentation.swift

```diff
@@ -1,6 +1,6 @@
 import UIKit
 
-open class PartialSheetPresentation: NSObject, UIViewControllerTransitioningDelegate {
+open class PartialSheetPresentation: NSObject, UIViewControllerTransitioningDelegate, Sendable {
```

Comment:
I believe this will fail on you. To be `Sendable` you must be a `final` class.

Reply:
when I build thumbprint I didn't get an error:

<img width="467" alt="Screenshot 2024-07-08 at 2 44 24 PM" src="https://github.com/thumbtack/thumbprint-ios/assets/113396903/bcaa3c62-7551-4036-95e3-05594e20f531">

Reply:
I'll take a look, maybe it only matters if we set targeted swift concurrency. But it's definitely a violation in *some* form of concurrency.

Reply:
:FYI we are inheriting from PartialSheetPresentation in WT

Reply:
I did set concurrency checking to complete with no issues:

<img width="456" alt="Screenshot 2024-07-09 at 1 09 33 PM" src="https://github.com/thumbtack/thumbprint-ios/assets/113396903/2d2cd9a0-2b61-484c-b872-957865347e55">